From 7c7f83f1f4cced06916ef9111027e88820b63ea0 Mon Sep 17 00:00:00 2001
From: Keith Smiley <keithbsmiley@gmail.com>
Date: Thu, 3 Mar 2022 20:30:49 -0800
Subject: [PATCH] [bazel] Add darwin_arm64 to list of cpus

This fixes bazel builds on Apple Silicon hosts

https://github.com/envoyproxy/envoy/issues/19916

Change-Id: I9651538bec8a98e4b350c5f8ee1a40ed91dff461
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3502848
Reviewed-by: Victor Gomes <victorgomes@chromium.org>
Commit-Queue: Victor Gomes <victorgomes@chromium.org>
Cr-Commit-Position: refs/heads/main@{#79364}
---
 AUTHORS        | 1 +
 bazel/defs.bzl | 1 +
 2 files changed, 2 insertions(+)

diff --git a/AUTHORS b/AUTHORS
index bef493120f..001981a334 100644
--- a/AUTHORS
+++ b/AUTHORS
@@ -146,6 +146,7 @@ Junha Park <jpark3@scu.edu>
 Junming Huang <kiminghjm@gmail.com>
 Kang-Hao (Kenny) Lu <kennyluck@csail.mit.edu>
 Karl Skomski <karl@skomski.com>
+Keith Smiley <keithbsmiley@gmail.com>
 Kevin Gibbons <bakkot@gmail.com>
 Kris Selden <kris.selden@gmail.com>
 Kyounga Ra <kyounga@alticast.com>
diff --git a/bazel/defs.bzl b/bazel/defs.bzl
index dee5e69cc4..ed7dea9581 100644
--- a/bazel/defs.bzl
+++ b/bazel/defs.bzl
@@ -389,6 +389,7 @@ def _v8_target_cpu_transition_impl(settings, attr):
         "aarch64": "arm64",
         "arm64-v8a": "arm64",
         "arm": "arm64",
+        "darwin_arm64": "arm64",
         "armeabi-v7a": "arm32",
         "s390x": "s390x",
         "riscv64": "riscv64",
-- 
2.35.1

